propolis/Cargo.toml

```diff
@@ -18,14 +18,20 @@ thiserror = "1.0"
 bhyve_api = { path = "../bhyve-api" }
 dladm = { path = "../dladm" }
 viona_api = { path = "../viona-api" }
-usdt = "0.1.11"
+usdt = { version = "0.1.18", default-features = false }
 tokio = { version = "1", features = ["full"] }
 futures = "0.3"
 slog = "2.7"
+serde = { version = "1" }
+serde_json = { version = "1", optional = true }
```

Comment:
So `serde_json` is required for the inline probes even though they aren't (currently) using `Serialize` types in their arguments?

Reply:
It's not actually, no. This was hanging around from local testing with a previous version of the `usdt` crate. Thanks for catching, it's removed in 91f3f61.